### DISCUSSION TITLE
Sharing picklist

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16502] [How to] [answered]
In VT it is pretty easy way of sharing the same picklist by changing fieldname columns to the the one where source values are (different module). It doesnt work in yetiforce. What will be the way of sharing the same picklist? I mean multi-picklist type
In vtiger it looks like below:
![image](https://user-images.githubusercontent.com/84110093/191283768-59a118d1-1f31-4711-91d2-72ce355e89ba.png)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
add a list field and enter an existing field name in the system name, it works
